cli.js

```diff
@@ -607,14 +605,8 @@ async function zkeyExportJson(params, options) {
     await bfj.write(zkeyJsonName, zKeyJson, {space: 1});
 }
 
-async function fileExists(file) {
-    return fs.promises.access(file, fs.constants.F_OK)
-        .then(() => true)
-        .catch(() => false);
-}
-
-// solidity genverifier [circuit_final.zkey] [verifier.sol]
-async function zkeyExportSolidityVerifier(params, options) {
+// zkey export verifier [circuit_final.zkey] [verifier.sol]
+async function zkeyExportVerifier(params, options) {
```

Comment:
This function needs to catch any error about unsupported protocols and show the user the plugin name and unsupported protocol name